### PR TITLE
Add Playwright UI Tests and Integration Tests for Search and Product Details Pages

### PR DESCRIPTION
**Pull request by DanielLimWeiEn**

### CODE REVIEW

File: .env.test
Comment (EvitanRelta) [inline .env.test:1]: remove this file. `.env` files shouldn't be commited

File: tests-examples/demo-todo-app.spec.cjs
Comment (EvitanRelta) [inline tests-examples/demo-todo-app.spec.cjs:1]: what this for?

**Review by EvitanRelta — CHANGES_REQUESTED** [review]
remove `.env.test`

File: tests-examples/demo-todo-app.spec.cjs
Comment (swxk19) [reply tests-examples/demo-todo-app.spec.cjs:1]: auto-generated by playwright upon installation. can be removed

**Pull request merged**